csharp/src/ProtocolBuffers/Google.Protobuf.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Google.Protobuf</id>
+    <title>Google Protocol Buffers C#</title>
+    <summary>C# runtime library for Protocol Buffers - Google's data interchange format.</summary>
+    <description>See project site for more info.</description>
+    <version>3.0.0-experimental</version>
```

Comment:
This would be better as 3.0.0-alpha1 or something like that - we can't reuse the same version number.